File: go-client-sdk/pkg/models/shared/codatdataintegritycontractssummarymatchcountsummary.go
package shared

type CodatDataIntegrityContractsSummaryMatchCountSummary struct {
	MatchPercentage *float64 `json:"matchPercentage,omitempty"`
	Matched         *int     `json:"matched,omitempty"`
	Total           *int     `json:"total,omitempty"`
	Unmatched       *int     `json:"unmatched,omitempty"`
}

File: python-client-sdk/src/sdk/models/shared/codat_dataintegrity_contracts_summary_matchcountsummary.py
from __future__ import annotations
import dataclasses
from dataclasses_json import Undefined, dataclass_json
from sdk import utils
from typing import Optional


@dataclass_json(undefined=Undefined.EXCLUDE)
@dataclasses.dataclass
class CodatDataIntegrityContractsSummaryMatchCountSummary:
    matched: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('matched'), 'exclude': lambda f: f is None }})
    match_percentage: Optional[float] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('matchPercentage'), 'exclude': lambda f: f is None }})
    total: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('total'), 'exclude': lambda f: f is None }})
    unmatched: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('unmatched'), 'exclude': lambda f: f is None }})
    